product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/12_pem_manage_probes/01_pem_custom_probes.mdx

```diff
@@ -13,7 +13,11 @@ Use the `Custom Probes` tab to create a new probe or modify an existing probe. A
 
 Use the `Show system probes?` switch to display the system probes on the `Custom Probes` tab.
 
-To modify an existing probe, click the `Edit` icon located to the left of a probe name. To create a new probe, click the `Add` icon in the upper-right corner of the tab.
+To modify an existing probe, click the `Edit` icon located to the left of a probe name.
+
+## Defining a New Probe
+
+To create a new probe, click the `Add` icon in the upper-right corner of the tab.
 
 <figure><img src="../../images/custom_probes_general.png" class="align-center" alt="Custom Probes - Add New Probe - General tab" /><figcaption aria-hidden="true"><em>Custom Probes - Add New Probe - General tab</em></figcaption></figure>
```

Comment:
The syntax for images need to be replaced with ! [Alternate_text](<path_to_image_file>/<image_filename.png>)